cmd/kubeturbo/app/kubeturbo_builder.go

```diff
@@ -254,7 +258,8 @@ func (s *VMTServer) Run() {
 		WithDiscoveryInterval(s.DiscoveryIntervalSec).
 		WithValidationTimeout(s.ValidationTimeout).
 		WithValidationWorkers(s.ValidationWorkers).
-		WithSccSupport(s.sccSupport)
+		WithSccSupport(s.sccSupport).
+		WithCAPINapespace(s.ClusterAPINamespace)
```

Comment:
Please correct typo

Reply:
Hi,
I don't see a typo. I agree we need to allow for the logic to handle other providers, although the Cluster API is not provider specific from coding standpoint.
I propose to address this when we know more.